mapie/tests/test_utils.py

```diff
@@ -132,3 +132,36 @@ def test_valid_shape_no_n_features_in(
         X=X_reg, cv="prefit", estimator=estimator
     )
     assert n_features_in == 10
+
+
+@pytest.mark.parametrize(
+    "alpha",
+    [
+        np.linspace(0.05, 0.95, 5),
+        [0.05, 0.95],
+        (0.05, 0.95),
+        np.array([0.05, 0.95])
+    ]
+)
+def test_valid_calcul_of_quantile(alpha: Any) -> None:
+    """Test that valid alphas raise no errors."""
+    n = 30
+    check_alpha_and_n_samples(alpha, n)
+
+
+@pytest.mark.parametrize(
+    "alpha",
+    [
+        np.linspace(0.05, 0.07),
+        [0.05, 0.07],
+        (0.05, 0.07),
+        np.array([0.05, 0.07])
+    ]
+)
+def test_invalid_calcul_of_quantile(alpha: Any) -> None:
+    """Test that alpha with 1/alpha < number of samples  raise errors."""
```

Comment:
Ne serait-ce pas si `1/alpha > number of samples` ?

Reply:
oui! corrigé